src/test/java/com/hillel/items_exchange/util/UserDtoCreatingUtil.java

```diff
@@ -111,4 +113,46 @@ public static UserDto getBuild(String username, String email, Boolean online, St
                 .phones(phones)
                 .build();
     }
+
+    public static UserChangePasswordDto createUserChangePasswordDtoWithCorrectData() {
+
+        return createUserChangePasswordDto(CORRECT_OLD_PASSWORD,
+                NEW_PASSWORD,
+                NEW_PASSWORD);
+    }
+
+    public static UserChangePasswordDto createUserChangePasswordDtoWithWrongOldPassword() {
+
+        return createUserChangePasswordDto(WRONG_OLD_PASSWORD,
+                NEW_PASSWORD,
+                NEW_PASSWORD);
+    }
+
+    public static UserChangePasswordDto createUserChangePasswordDtoWithWrongPasswordConfirmation() {
+
+        return createUserChangePasswordDto(CORRECT_OLD_PASSWORD,
```

Comment:
You shouldn't create a new method every time when you want to invoke current method. Just use it.

Reply:
done